packages/rocketchat-ui-account/account/accountProfile.html

```diff
@@ -59,6 +59,9 @@
 				<div class="submit">
 					<button class="button"><i class="icon-send"></i><span>{{_ "Save_changes"}}</span></button>
 				</div>
+				<div class="logoutOthers">
+					<button class="button">Logout Others</button>	
```

Comment:
Better than this, just use the Blaze helper, like this:
`{{_ "Logout_Others"}}`